zktest/test/java/org/zkoss/zktest/zats/test2/F85_ZK_3846Test.java

```diff
@@ -0,0 +1,47 @@
+/* F85_ZK_3846.java
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Thu Jun 07 15:45:28 CST 2018, Created by charlesqiu
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.zkoss.zktest.zats.WebDriverTestCase;
+import org.zkoss.zktest.zats.ztl.JQuery;
+
+public class F85_ZK_3846Test extends WebDriverTestCase {
```

Comment:
Try `Clients.reloadMessages` for testing de_DE and non de_DE

Reply:
OK